postile/postile.py

```diff
@@ -227,6 +253,9 @@ def main():
     if args.cors:
         CORS(app)
 
+    # add static route for the favicon 
+    app.static('/favicon.ico', str(here / 'static/favicon.ico'))
```

Comment:
Maybe we could ask Sylvain to make a logo for postile, as he seems to get used to making logo for OSS projects these times ? (he did the proj logo recently).

Reply:
yeah good idea, @sylvainbeo interested ?

Reply:
@autra @ldgeo Sure !